src/phasorpy/phasor.py

```diff
@@ -0,0 +1,507 @@
+"""Manipulate phasor  data
+
+The ``phasorpy.phasor`` module provides functions to:
+    - transform phasor to polar coordinates
+    - calibration of phasor coordinates
+    - compute phasor coordinates from lifetime values
+
+
+Lifetime values for known fluorophores can be obtained from the ISS website:
+https://iss.com/resources#lifetime-data-of-selected-fluorophores.
+
+"""
+
+from __future__ import annotations
+
+__all__ = [
+    "phasor_calibration",
+    "polar_from_reference_phasor",
+    "polar_from_reference",
+    "phasor_to_polar",
+    "phasor_from_lifetime",
+    "phasor_center",
+    "_mean",
+    "_spatial_median",
+    "_geometric_median",
+]
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._typing import Any, NDArray, ArrayLike, Literal
+
+import math
+
+import numpy
+
+
+def phasor_calibration(
+    real: ArrayLike,
+    imag: ArrayLike,
+    phase0: ArrayLike = 0.0,
+    modulation0: ArrayLike = 1.0,
+    /,
+) -> tuple[ArrayLike, ArrayLike]:
```

Comment:
I had to change it to ArrayLike because MyPy was returning an error because of this part:

`    if numpy.all(phi0 == 0) and numpy.all(mod0 == 1):
        return real, imag`

This way if the input is to remain unchanged, the returned elements are exactly the same as the input. I don't know if it is the best way to  go. If you consider better to always return the same type I can force the conversion to numpy array.

Reply:
MyPy is right. But I think `phi0 ` and `mod0` should be converted to arrays before passing them to `numpy.all(... == 0)` anyway. Also, `phi0==0` and `mod0==1` is an exceptional case, which might not be worth?